include/zephyr/dt-bindings/clock/kinetis_sim.h

```diff
@@ -15,6 +15,9 @@
 #define KINETIS_SIM_DMAMUX_CLK  KINETIS_SIM_BUS_CLK
 #define KINETIS_SIM_DMA_CLK     KINETIS_SIM_CORESYS_CLK
 #define KINETIS_SIM_SIM_SOPT7   7
+#define KINETIS_SIM_OSCERCLK    8
+#define KINETIS_SIM_MCGIRCLK    12
```

Comment:
these changes seem to be in the wrong commit.

Reply:
It is intentional. The defines can be used to select clock source for LPUART in devicetree. Do you prefer to have this change in another commit?

Reply:
Adding here- please do move this change to another commit

Reply:
Moved to separate commit.